packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/templates/pack-xdsl-modem-templates.html

```diff
@@ -10,11 +10,15 @@
         <oui-message
             type="success"
             data-ng-if="$ctrl.message.type === 'success'"
-            >{{$ctrl.message.detail}}</oui-message
-        >
-        <oui-message type="error" data-ng-if="$ctrl.message.type === 'error'"
-            >{{$ctrl.message.detail}}</oui-message
         >
+            {{$ctrl.message.detail}}
```

Comment:
```suggestion
<span data-ng-bind="$ctrl.message.detail"></span>
```

Reply:
You have done for error message but the modification isn't done for success message. you have to use data-ng-bind